helm/data/values/prod/values.yaml

```diff
@@ -1,12 +1,6 @@
-imagePullPolicy: IfNotPresent
-
 deployableNamespaces:
   - whoami
```

Comment:
I suggest we remove whoami from prod. If we as developers needs to test we can use dev

Reply:
Again, just for consistency with the `im-manager`, [where we have the `whoami` group](https://github.com/dhis2-sre/im-manager/blob/master/helm/data/values/prod/values.yaml#L1-L3).